tests/sensu-client-windows.pp

```diff
@@ -0,0 +1,9 @@
+# Note: Check http://repositories.sensuapp.org/msi/ for the latest version.
+class { '::sensu':
+  version           => '0.29.0-11',
+  rabbitmq_password => 'correct-horse-battery-staple',
+  rabbitmq_host     => '192.168.56.10',
+  rabbitmq_vhost    => '/sensu',
+  subscriptions     => 'all',
+  client_address    => $facts['networking']['ip'],
```

Comment:
@ghoneycutt Please LMK if `$facts['networking']['ip']` is resolving to the incorrect `10.0.2.15` address for you.  Digging into this, the networking fact returns the IP of the primary interface, and the opentable wind2012r2 VM has the second network interface marked as primary.  With this information I expect `$facts['networking']['ip']` to be robust.

I've also confirmed the IP shows up as 192.168.56.15 at http://localhost:3000/#/clients